Userland/Libraries/LibArchive/Tar.h

```diff
@@ -89,21 +90,21 @@ static void set_octal_field(char (&field)[N], TSource&& source)
 class [[gnu::packed]] TarFileHeader {
 public:
     StringView filename() const { return get_field_as_string_view(m_filename); }
-    mode_t mode() const { return get_field_as_integral(m_mode); }
-    uid_t uid() const { return get_field_as_integral(m_uid); }
-    gid_t gid() const { return get_field_as_integral(m_gid); }
+    mode_t mode() const { return MUST(get_field_as_integral(m_mode)); }
+    uid_t uid() const { return MUST(get_field_as_integral(m_uid)); }
+    gid_t gid() const { return MUST(get_field_as_integral(m_gid)); }
     // FIXME: support 2001-star size encoding
-    size_t size() const { return get_field_as_integral(m_size); }
-    time_t timestamp() const { return get_field_as_integral(m_timestamp); }
-    unsigned checksum() const { return get_field_as_integral(m_checksum); }
+    size_t size() const { return MUST(get_field_as_integral(m_size)); }
+    time_t timestamp() const { return MUST(get_field_as_integral(m_timestamp)); }
+    ErrorOr<unsigned> checksum() const { return get_field_as_integral(m_checksum); }
     TarFileType type_flag() const { return TarFileType(m_type_flag); }
     StringView link_name() const { return { m_link_name, strlen(m_link_name) }; }
     StringView magic() const { return get_field_as_string_view(m_magic); }
     StringView version() const { return get_field_as_string_view(m_version); }
     StringView owner_name() const { return get_field_as_string_view(m_owner_name); }
     StringView group_name() const { return get_field_as_string_view(m_group_name); }
-    int major() const { return get_field_as_integral(m_major); }
-    int minor() const { return get_field_as_integral(m_minor); }
+    int major() const { return MUST(get_field_as_integral(m_major)); }
+    int minor() const { return MUST(get_field_as_integral(m_minor)); }
```

Comment:
All of these should probably pass through parsing errors as well instead of using `MUST`, otherwise we'll just get the same fuzzing failures with those next.